DSESwift/DataStorageEngine.swift

```diff
@@ -31,12 +30,12 @@ public class DataStorageEngine{
     // reference to the datasyncer
     var dataSyncer = DataSyncer()
```

Comment:
@orhan89 hmm, why? DataSyncer is used by DataStorageEngine class, so it needs to be internal, right?